fastlane_bot/modes/triangle_multi_complete.py

```diff
@@ -85,12 +85,24 @@ def get_all_relevant_pairs_info(CCm, all_relevant_pairs, carbon_v1_forks):
         pair_curves = CCm.bypair(pair)
         carbon_curves = []
         non_carbon_curves = []
+        base_direction_one = []
+        base_direction_two = []
+        base_direction_pair = pair_curves[0].pair
         for x in pair_curves:
             if x.params.exchange in carbon_v1_forks:
                 carbon_curves += [x]
```

Comment:
What is this used for?

Reply:
The base direction is used to sort carbon orders into the groups of which token (in the pair) is being sold. This way all the orders selling the same asset are grouped together.
This downstream processes are dependent on this.

Reply:
I was referring to `carbon_curves`. Is it still used anywhere?